symphony/content/content.publish.php

```diff
@@ -1668,7 +1668,7 @@ private function prepareAssociationsDrawer($section)
                         // get associated entries if entry exists,
                         if ($entry_id) {
                             $relation_field = FieldManager::fetch($as['child_section_field_id']);
-                            $entry_ids = $relation_field->findParentRelatedEntries($as['parent_section_field_id'], $entry_id);
+                            $entry_ids = $relation_field->findParentRelatedEntries($as['child_section_field_id'], $entry_id);
```

Comment:
You mean this was already not working for some fields in `2.6.x`? As it works with our first party fields, isn't this more of an issue within the respective fields then?